devicetypes/erocm123/inovelli-2-channel-smart-plug.src/inovelli-2-channel-smart-plug.groovy

```diff
@@ -252,13 +252,19 @@ private channelNumber(String dni) {
 }
 private void createChildDevices() {
 	state.oldLabel = device.label
-	for (i in 1..2) {
-		addChildDevice("Switch Child Device", "${device.deviceNetworkId}-ep${i}", null, [completedSetup: true, label: "${device.displayName} (CH${i})",
-																						 isComponent: true, componentName: "ep$i", componentLabel: "Channel $i"
-		])
+	try {
+		for (i in 1..2) {
+			addChildDevice("Switch Child Device", "${device.deviceNetworkId}-ep${i}", null, [completedSetup: true, label: "${device.displayName} (CH${i})",
+				isComponent: false, componentName: "ep$i", componentLabel: "Channel $i"
```

Comment:
We don't want to do this. The motivation comes from wanting to be able to re-label the device, but not treating it as a component has other implications for One App and the API. There is this pull request that would allow components to be re-labeled: https://github.com/PhysicalGraph/data-management/pull/4770